File: devhub_v2/backend/data/projects/745298e1-5589-461e-a353-a9fc696b0e62/backend/prisma/seed.ts
import { PrismaClient } from '@prisma/client';

const prisma = new PrismaClient();

async function main() {
  // Clear existing data
  await prisma.appointment.deleteMany();
  await prisma.doctor.deleteMany();
  await prisma.patient.deleteMany();

  // Create Patient
  const patient = await prisma.patient.create({
    data: {
      name: 'Alex Johnson',
      email: 'alex.johnson@example.com',
    },
  });

  // Create Doctors
  const doctors = await Promise.all([
    prisma.doctor.create({ data: { name: 'Dr. Sarah Jenkins', specialty: 'Cardiology', rating: 4.9, imageUrl: 'https://picsum.photos/seed/dr1/200' } }),
    prisma.doctor.create({ data: { name: 'Dr. Michael Chen', specialty: 'Neurology', rating: 4.8, imageUrl: 'https://picsum.photos/seed/dr2/200' } }),
    prisma.doctor.create({ data: { name: 'Dr. Emily Rodriguez', specialty: 'Pediatrics', rating: 4.9, imageUrl: 'https://picsum.photos/seed/dr3/200' } }),
    prisma.doctor.create({ data: { name: 'Dr. James Wilson', specialty: 'General Practice', rating: 4.7, imageUrl: 'https://picsum.photos/seed/dr4/200' } }),
    prisma.doctor.create({ data: { name: 'Dr. Olivia Taylor', specialty: 'Dermatology', rating: 4.9, imageUrl: 'https://picsum.photos/seed/dr5/200' } }),
  ]);

  // Create Appointments
  await prisma.appointment.create({
    data: {
      date: new Date(new Date().getTime() + 24 * 60 * 60 * 1000), // Tomorrow
      status: 'SCHEDULED',
      doctorId: doctors[0].id,
      patientId: patient.id,
    },
  });

  await prisma.appointment.create({
    data: {
      date: new Date(new Date().getTime() - 48 * 60 * 60 * 1000), // 2 days ago
      status: 'COMPLETED',
      doctorId: doctors[3].id,
      patientId: patient.id,
    },
  });

  console.log('Database seeded successfully!');
}

main()
  .catch((e) => {
    console.error(e);
    process.exit(1);
  })
  .finally(async () => {
    await prisma.$disconnect();
  });

File: devhub_v2/backend/data/chat-checkpoints/f500f473-e47e-42b9-9d03-3991afc3c245/7aa81a3f0ee74199a6a0ba1655d602d0/snapshot/backend/seed.py
import os
import django

os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'omnia_backend.settings')
django.setup()

from api.models import Doctor, Service

def seed():
    Doctor.objects.all().delete()
    Service.objects.all().delete()

    Doctor.objects.create(name="Dr. Sarah Jenkins", specialty="Cardiology", experience_years=12, image_url="https://images.unsplash.com/photo-1559839734-2b71ea197ec2?auto=format&fit=crop&q=80&w=300&h=300")
    Doctor.objects.create(name="Dr. Michael Chen", specialty="Dermatology", experience_years=8, image_url="https://images.unsplash.com/photo-1612349317150-e413f6a5b16d?auto=format&fit=crop&q=80&w=300&h=300")
    Doctor.objects.create(name="Dr. Emily Rodriguez", specialty="Pediatrics", experience_years=15, image_url="https://images.unsplash.com/photo-1594824476967-48c8b964273f?auto=format&fit=crop&q=80&w=300&h=300")
    Doctor.objects.create(name="Dr. James Wilson", specialty="Orthopedics", experience_years=20, image_url="https://images.unsplash.com/photo-1622253692010-333f2da6031d?auto=format&fit=crop&q=80&w=300&h=300")
    Doctor.objects.create(name="Dr. Olivia Parker", specialty="Neurology", experience_years=10, image_url="https://images.unsplash.com/photo-1559839734-2b71ea197ec2?auto=format&fit=crop&q=80&w=300&h=300")
    Doctor.objects.create(name="Dr. William Davis", specialty="Oncology", experience_years=18, image_url="https://images.unsplash.com/photo-1612349317150-e413f6a5b16d?auto=format&fit=crop&q=80&w=300&h=300")
    Doctor.objects.create(name="Dr. Sophia Martinez", specialty="Psychiatry", experience_years=14, image_url="https://images.unsplash.com/photo-1594824476967-48c8b964273f?auto=format&fit=crop&q=80&w=300&h=300")
    Doctor.objects.create(name="Dr. Alexander White", specialty="General Surgery", experience_years=22, image_url="https://images.unsplash.com/photo-1622253692010-333f2da6031d?auto=format&fit=crop&q=80&w=300&h=300")

    Service.objects.create(name="Video Consultation", description="Consult with top doctors from the comfort of your home.", icon_name="Video")
    Service.objects.create(name="In-Clinic Visit", description="Book appointments at our state-of-the-art facilities.", icon_name="Building2")
    Service.objects.create(name="Lab Tests", description="Home sample collection for all major pathology tests.", icon_name="TestTube")
    Service.objects.create(name="Medicines", description="Get medicines delivered to your doorstep within 2 hours.", icon_name="Pill")
    Service.objects.create(name="Health Checkups", description="Comprehensive health packages for you and your family.", icon_name="ShieldPlus")
    Service.objects.create(name="Dental Care", description="Expert dental services and treatments.", icon_name="Star")

    print("Database seeded successfully!")

if __name__ == '__main__':
    seed()
